### ISSUE TITLE
how about proguard settings?

### ISSUE THREAD

**Issue by wispy316** [oguzbabaoglu/butterfork#3]
Which one I should add into proguard.txt.
Is there a example?

**Comment by oguzbabaoglu** [comment]
Same as ButterKnife

```
-keep class butterfork.** { *; }
-dontwarn butterfork.internal.**
-keep class **$$ViewBinder { *; }

-keepclasseswithmembernames class * {
    @butterfork.* <fields>;
}

-keepclasseswithmembernames class * {
    @butterfork.* <methods>;
}
```
I'll add a `consumerProguardFiles` option to gradle for next release

**Comment by oguzbabaoglu** [comment]
Fixed with update to ButterKnife 8.0.1
